HelpSource/Classes/HistoryGui.schelp

```diff
@@ -19,6 +19,99 @@ DEFINITIONLIST::
 ## listview || shows either all code::lineShorts::, or the filtered code::lineShorts::; selecting them in listview makes them come appear in post-doc window.
 ::
 
+first example:
+code::
+(
+h = History.new.lines_([
+	[0, \me, "1+2"],
+	[1, \me, "3+5"], [1.234, \you, "q = q ? ();"],
+	[3, \her, "\"Herstory\".speak"]
+]);
+g = h.makeWin(0@20);
+g.findDoc;
+g.postDoc(2);
+)
+::
+
+ClassMethods::
+
+method::docHeight
+get and set height of textdoc/window opened by HistoryGui
```

Comment:
these kinds of descriptions feel unnecessarily terse. would you consider expanding them to something like, "Get and set the height of the textdoc or window opened by HistoryGui"? also, what is a "textdoc"? I've never seen that term in documentation before

Reply:
done